packages/graphql_flutter/pubspec.lock

```diff
@@ -91,10 +91,10 @@ packages:
   graphql:
     dependency: "direct main"
     description:
-      name: graphql
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "1.0.1-beta.8"
+      path: "../graphql"
```

Comment:
Do we need to regen this lock file so it doesn't have a relative dependency?

Reply:
May be we should just add the pubspec.lock files to gitignore https://dart.dev/guides/libraries/private-files#pubspeclock

Reply:
will ignore as suggested